website/docs/cli/commands/modules.mdx

```diff
@@ -0,0 +1,17 @@
+---
+page_title: 'Command: modules'
+description: >-
+  The terraform modules command prints information about all the declared
+  modules in Terraform configuration.
+---
+
+# Command: modules
+
+The `terraform modules` command provides a holistic view of all Terraform
+modules and their resolved versions that have been declared in Terraform
+configuration for the current working directory.
+
```

Comment:
Can you add context on _when_ someone would want to run this? Could be helpful to touch on using this instead of `modules.json`.